public/index.css

```diff
@@ -0,0 +1,154 @@
+html, body {
```

Comment:
fix indentations please, it's hard to read this file

Reply:
i took template from an internet, this styles weren't written by me